src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy

```diff
@@ -463,17 +440,6 @@ class AutoRerouteSpec extends HealthCheckSpecification {
                 !(it.details =~ /Reason: ISL .* status become ACTIVE/)
             }.isEmpty()
         }
-
-        cleanup:
-        if (isSwDeactivated) {
```

Comment:
Nice to have, please ignore it if you want. There is "isSwDeactivated = false" defined on line 435, but it is not used anymore